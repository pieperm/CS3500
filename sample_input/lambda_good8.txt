(let* ( (f (lambda (x) (+ 1 x)))
      )
 (progn
  (f "foo")
  (f 100)
 )
)
