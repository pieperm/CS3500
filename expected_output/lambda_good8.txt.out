
___Entering new scope...


___Entering new scope...

___Adding x to symbol table

___Exiting scope...

___Adding f to symbol table

___Exiting scope...

EXPR type is: INT
---- Completed parsing ----


Bye!
